scripts/sdk/build.sh

```diff
@@ -10,13 +10,7 @@ mkdir target/paths
 cp doc/SDK.md target/README.md
 pushd target
 pushd asset-build/assets
-mv main.*.js ../../main.js
-mv index.*.css ../../style.css
-mv download-sandbox.*.html ../../download-sandbox.html
-rm *.js *.wasm
-mv ./* ../../
+rm !(main).js *.wasm
```

Comment:
We want to remove all `*.js` files except for `main.js`

Reply:
This needs `shopt -s extglob` to enable extended globs.

Seems to work fine with git bash on Windows by default but had issues on macOS

```
./scripts/sdk/build.sh: line 13: syntax error near unexpected token `('
./scripts/sdk/build.sh: line 13: `rm !(main).js *.wasm'
```